frontend/public/style/_vars.scss

```diff
@@ -73,6 +73,9 @@ $font-size-base: 13px;
 // Patternfly defaults to 40px, which leaves a lot more whitespace than we're used to
 $grid-gutter-width: 30px;
 
+$co-namespace-selector-desktop: 40px;
```

Comment:
The existing convention with vars is to only put them in the vars partial if they are used by more than one partial.  In other words, move these to frontend/public/components/_dropdown.scss